package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "format-graphql": "^1.5.0"
+  }
+}
```

Comment:
we don't need this either. You can install `format-graphql` globally on your machine instead